lib/_internal/_reduce/index.js

```diff
@@ -1,4 +1,4 @@
-function reduceIterable(cb, acc, gen) {
+const reduceIterable = (cb, acc, gen) => {
```

Comment:
👌 this is good helpers are still have names, because stack trace will be readable

Reply:
Yeah, I try to care about it as much as about a bundle size! :)